interfaces/apparmor/apparmor_test.go

```diff
@@ -70,6 +73,22 @@ apparmor_parser output:
 		"--replace --write-cache -O no-expr-simplify --cache-loc=/var/cache/apparmor /path/to/snap.samba.smbd"})
```

Comment:
This context for the diff shows a hard-coded --cache-loc. Please adjust it and any other uses of --cache-loc. With those changes, LGTM.